### ISSUE TITLE
[Accessibility]: Impossible to select a range of lines using screen reader controls

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36527]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34448

### Problem description

A person who uses a mouse without screen reader can click on a line, hold down shift and click on another line -- this set of steps selects a contiguous sequence of lines. The URL is updated to reflect the sequence of lines, which can be shared with someone to (say) showcase a code example.

It does not seem possible to select multiple lines in such a way with screen reader controls.

### Expected behavior

There should be some way to select multiple lines. When multiple lines are selected, the screen reader should announce the range of lines that is selected.

### Additional details

Related problem where selection for a single line doesn't work:
- https://github.com/sourcegraph/sourcegraph/issues/36526

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by varungandhi-src** [comment]
Related problem for mobile: https://github.com/sourcegraph/sourcegraph/issues/36635
